### PR TITLE
[WFLY-9777] clean testsuite debris in git-trackable directories.

### PR DESCRIPTION
**Pull request by soul2zimate**
https://issues.jboss.org/browse/WFLY-9777 clean testsuite debris in git-trackable directories.

### CODE REVIEW

**Comment by jmesnil** [comment]
retest this please

**Review by dmlloyd — APPROVED** [review]

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/usertransaction/UserTransactionAccessTestCase.java [hunk 1]
```diff
@@ -46,7 +47,7 @@
     public static JavaArchive createDeployment() {
         final JavaArchive jar = ShrinkWrap.create(JavaArchive.class, MODULE_NAME + ".jar");
         jar.addPackage(UserTransactionAccessTestCase.class.getPackage());
-        jar.as(ZipExporter.class).exportTo(new File(".", jar.getName()), true);
+        jar.as(ZipExporter.class).exportTo(new File(TestSuiteEnvironment.getTmpDir(), jar.getName()), true);
```
Comment (stuartwdouglas) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/usertransaction/UserTransactionAccessTestCase.java]: I think this like should have just been deleted, there is no reason to export the archive

File: testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestCase.java [hunk 2]
```diff
@@ -268,7 +269,7 @@ private String getClientJarPaths() throws IOException {
         jar.addManifest()
                 .addAsManifestResource(WSTrustTestCase.class.getPackage(), "META-INF/clientKeystore.properties", "clientKeystore.properties")
                 .addAsManifestResource(WSTrustTestCase.class.getPackage(), "META-INF/clientstore.jks", "clientstore.jks");
-        File jarFile = new File("jaxws-samples-wsse-policy-trust-client.jar");
+        File jarFile = new File(TestSuiteEnvironment.getTmpDir(), "jaxws-samples-wsse-policy-trust-client.jar");
```
Comment (stuartwdouglas) [inline testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestCase.java]: Same here.

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/usertransaction/UserTransactionAccessTestCase.java (hunk 1 above)
Comment (soul2zimate) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/usertransaction/UserTransactionAccessTestCase.java]: deleted.

File: testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestCase.java (hunk 2 above)
Comment (soul2zimate) [reply testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestCase.java]: it's used to create an URLClassLoader with these security files as resource and put into setContextClassLoader by TestWatcher, then later it retrieves back from [WSTrustTestUtils static methods](https://github.com/wildfly/wildfly/blob/6b61a6003f704221f66dcd9f418bcb7af88fb9ab/testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestUtils.java#L264) for client env setup.

**Comment by stuartwdouglas** [comment]
Retest this please

**Review by stuartwdouglas — APPROVED** [review]

**Comment by stuartwdouglas** [comment]
Retest this please

**Comment by soul2zimate** [comment]
failed ConcurrentCoarseWebFailoverTestCase.testGracefulSimpleFailover in Windows CI does not seem related.

**Pull request merged**